cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -428,6 +433,11 @@ In order to configure CDAP Master for Kerberos authentication:
       process. The string "_HOST" will be substituted with the local hostname.</description>
     </property>
 
+- The ``/cdap`` directory needs to be owned by the ``<cdap-principal>``; you can set
```

Comment:
Same as with the secure HBase command, we should frame this as "run as the hdfs user".  So maybe change the last part to: "you can set that by running the following command as the ``hdfs`` user (substituting for ``<cdap-principal>``)"

Then remove the "sudo -u hdfs" from the command.